truss/_integration-tests/cli/test-service-definitions/1-basic/basic.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+package TEST;
+
+import "google/api/annotations.proto";
+
+service Basic {
+  rpc GetBasic (BasicTypeRequest) returns (BasicTypeResponse) {
+    option (google.api.http) = {
+      get: "/1"
+    };
+  }
+
+
+  rpc PostBasic (BasicTypeRequest) returns (BasicTypeRequest) {
+    option (google.api.http) = {
+      post: "/2"
+      body: "*"
+    };
+  }
+}
+
+message BasicTypeRequest {
+  double A = 1;
+  float B = 2;
+  int32 C = 3;
+  int64 D = 4;
+  uint32 E = 5;
+  uint64 F = 6;
+  sint32 G = 7;
+  sint64 H = 8;
+  fixed32 I = 9;
+  fixed64 J = 10;
+  sfixed32 K = 11;
+  bool L = 12;
+  string M = 13;
```

Comment:
No complaints, I'm just really happy to have such thorough test cases to go work with!